pkg/logger/charmbracelet.go

```diff
@@ -0,0 +1,147 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/charmbracelet/lipgloss"
+	log "github.com/charmbracelet/log"
+	"github.com/cloudposse/atmos/pkg/ui/theme"
+)
+
+const TraceLevel log.Level = log.DebugLevel - 1
+
+var helperLogger *log.Logger
+
+func init() {
+	helperLogger = GetCharmLogger()
+}
+
+// GetCharmLogger returns a pre-configured Charmbracelet logger with Atmos styling.
+func GetCharmLogger() *log.Logger {
+	styles := getAtmosLogStyles()
+	logger := log.New(os.Stderr)
+	logger.SetStyles(styles)
+	return logger
+}
+
+// GetCharmLoggerWithOutput returns a pre-configured Charmbracelet logger with custom output.
+func GetCharmLoggerWithOutput(output *os.File) *log.Logger {
+	styles := getAtmosLogStyles()
+	logger := log.New(output)
+	logger.SetStyles(styles)
+	return logger
+}
+
+// getAtmosLogStyles returns custom styles for the Charmbracelet logger using Atmos theme colors.
+func getAtmosLogStyles() *log.Styles {
+	styles := log.DefaultStyles()
+
+	const (
+		paddingVertical   = 0
+		paddingHorizontal = 1
+	)
+
+	configureLogLevelStyles(styles, paddingVertical, paddingHorizontal)
+	configureKeyStyles(styles)
+
+	return styles
+}
+
+// configureLogLevelStyles configures the styles for different log levels.
+func configureLogLevelStyles(styles *log.Styles, paddingVertical, paddingHorizontal int) {
+	const (
+		errorLevelLabel = "ERROR"
+		warnLevelLabel  = "WARN"
+		infoLevelLabel  = "INFO"
+		debugLevelLabel = "DEBUG"
```

Comment:
Let's stick to 4 characters, or the scrolling text is jarring. 
```suggestion
		debugLevelLabel = "DEBU"
```